src/core.js

```diff
@@ -62,7 +62,16 @@ let doc = null;
  * @template T
  */
 const patchFactory = function(run) {
-  return function(node, fn, data) {
+  /**
+   * TODO(moz): These annotations won't be necessary once we switch to Closure
```

Comment:
Is there roughly a timeframe when this will be resolved? Is there something about the current state of NTI that makes its current state not very friendly?

Just want to make sure that this doesn't linger here forever.